server/src/Server.js

```diff
@@ -21,6 +21,11 @@ if (settings.DEV_MODE) {
     });
 }
 
+app.get('/map/:z/:x/:y', function (req, res) {
+    const {z, x, y} = req.params;
+    res.sendFile(path.join(__dirname, `../map/leaflet-map/${z}/${x}/${y}`));
```

Comment:
I think sendFile needs an error handler as panning the map over an area that has no tile images (such as hidden areas outside of the current visible map, or out of the map bounds) causes a spam of file not found error messages in the terminal.

Reply:
Working on a fix 💯